app/models/setting/general.rb

```diff
@@ -26,6 +26,7 @@ def self.default_settings
       self.set('lab_features', N_("Whether or not to show a menu to access experimental lab features (requires reload of page)"), false, N_('Show Experimental Labs')),
       self.set("append_domain_name_for_hosts", N_("Foreman will append domain names when new hosts are provisioned"), true, N_("Append domain names to the host")),
       self.set('outofsync_interval', N_("Duration in minutes after servers are classed as out of sync."), 30, N_('Out of sync interval')),
+      self.set('instance_id', N_("Foreman instance ID, uniquely identifies this Foreman instance."), Foreman.uuid, N_('Foreman UUID')),
```

Comment:
This changes the value with every restart, since settings is set to nil and we use the default value. We may want to generate it in migration.

Reply:
It doesn't - on first run it creates a record in `settings` table, and the value will stay there. At least that's what I see in my dev DB.

Reply:
that didn't happen on my DB when I tested, I think we don't save any value to DB until the setting is changed, nil means use the default value which in this case change with every restart

Reply:
@ares, my bad, didn;t count the parameters right. Now I set `value` instead of `default_value` which is not overridden after multiple `rails s` invocations.